src/main/java/com/example/h2/SweggerConfigeration.java

```diff
@@ -0,0 +1,35 @@
+package com.example.h2;
+
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+
+@Configuration
+public class SweggerConfigeration {
+
+    @Bean
+//    PORT : http://localhost:9090/swagger-ui/index.html
```

Comment:
What is this PORT ? Here

Reply:
these is for swagger link to handle the request directly